sumologic/sumologic_sources.go

```diff
@@ -74,7 +74,7 @@ func resourceSumologicSource() *schema.Resource {
 			"timezone": {
 				Type:     schema.TypeString,
 				Optional: true,
-				Default:  "Etc/UTC",
+				Default:  nil,
```

Comment:
this needs to be set to empty string `""`

Reply:
Updated nil to ""